.github/actions/cdn_assets/action.yml

```diff
@@ -2,10 +2,6 @@ name: Download CLI assets
 description: Download CLI assets from Grafbase CDN
 
 inputs:
-  working-directory:
-    description: Where to run
-    required: false
-    default: cli
```

Comment:
Why was this moved to each individual step?

Reply:
It was removed as an input value, since we're not actually passing this input from anywhere.